packages/components/src/box-control/index.js

```diff
@@ -94,7 +95,7 @@ export default function BoxControl( {
 	const handleOnReset = () => {
 		const initialValues = DEFAULT_VALUES;
 
-		onChange( initialValues );
+		onReset();
```

Comment:
With this change, when `onReset` isn't provided it will become noop which is change in the behavior. Should it run all logic instead for backward compatibility?

Reply:
This was kind of tricky, so I changed the reset function to return true so we could see if it's set or not. Let me know if you can think of a better way to do it.